### PR TITLE
Created .editorconfig

### LINKED ISSUE #25: Add .editorconfig
Add a .editorconfig file.

### PR DESCRIPTION
**Pull request by AmanRaj1608**
Fixes #25 

### Description

Added .editorconfig as per project details

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,16 @@
+root = true
+
+[*]
+end_of_line = lf
+insert_final_newline = true
```
Comment (yashLadha) [inline .editorconfig]: put `indent_style` in root and remove from other places

File: .editorconfig [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+[*.{js,py}]
+charset = utf-8
+indent_size = 2
+
+[Makefile]
+indent_style = space
+
+[{package.json,.shrinkwrap.yaml}]
```
Comment (yashLadha) [inline .editorconfig]:
```suggestion
[*.{json,yaml}]
```

File: .editorconfig [hunk 3]
```diff
@@ -0,0 +1,16 @@
+root = true
+
+[*]
+end_of_line = lf
+insert_final_newline = true
+
+[*.{js,py}]
+charset = utf-8
```
Comment (yashLadha) [inline .editorconfig]: Put inside `root`

**Comment by AmanRaj1608** [comment]
Added changes as asked

**Comment by AmanRaj1608** [comment]
Added changes as asked

File: .editorconfig (hunk 3 above)
Comment (AmanRaj1608) [reply .editorconfig]: moved to root as you asked

File: .editorconfig (hunk 2 above)
Comment (AmanRaj1608) [reply .editorconfig]: Done

File: .editorconfig [hunk 4]
```diff
@@ -0,0 +1,14 @@
+root = true
+
+[*]
+end_of_line = lf
+insert_final_newline = true
+charset = utf-8
+indent_style = space
+indent_size = 2
+
+[{createmail.js}]
```
Comment (aashutoshrathi) [inline .editorconfig]: No need of `[{createmail.js}]`.

File: .editorconfig [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+indent_size = 4
+
+[*.{md}]
+indent_size = 4
```
Comment (aashutoshrathi) [inline .editorconfig]: Remember when you write `end_of_line = lf` under the root, you want to say that file should be ending with an empty new line feed.
So, this file should also follow the same

Comment (AmanRaj1608) [reply .editorconfig]: Noted.
Thanks

File: .editorconfig (hunk 4 above)
Comment (AmanRaj1608) [reply .editorconfig]: But createmail.js contains 4 spaces.

Comment (AmanRaj1608) [reply .editorconfig]: Still removed

File: .editorconfig [hunk 6]
```diff
@@ -0,0 +1,12 @@
...
+[*]
+end_of_line = lf
+insert_final_newline = true
+charset = utf-8
+indent_style = space
+indent_size = 2
+
+[*.{md}]
+indent_size = 4
+
```
Comment (aashutoshrathi) [inline .editorconfig]: Is this empty? 😕

**Comment by AmanRaj1608** [comment]
Is it fine like this. If not please help a little 😣

**Comment by singhpratyush** [comment]
This being done, #38 would be the next thing to resolve in this direction.

**Comment by aashutoshrathi** [comment]
Sure @singhpratyush 😄

**Pull request merged**